common/lib/xmodule/xmodule/poll_module.py

```diff
@@ -215,15 +215,15 @@ def definition_from_xml(cls, xml_object, system):
 
     def definition_to_xml(self, resource_fs):
         """Return an xml element representing to this definition."""
-        poll_str = u'<{tag_name}>{text}</{tag_name}>'.format(
+        poll_str = HTML('<{tag_name}>{text}</{tag_name}>').format(
             tag_name=self._tag_name, text=self.question)
         xml_object = etree.fromstring(poll_str)
         xml_object.set('display_name', self.display_name)
 
         def add_child(xml_obj, answer):
             # Escape answer text before adding to xml tree.
             answer_text = six.text_type(Text(answer['text']))
```

Comment:
```suggestion
            answer_text = str(answer['text']))
```

I think that since we now are using HTML in the next line, it is not needed anymore the escape in here.

At this moment this is failing because it is double escaping this.

Could you rebase and try this one @leomartinez2019 ?

Reply:
Probably we also can remove the str(), but I just added that in order to be compatible with the output of six.text_type

Reply:
On the other hand, if we remove the usage of six here, we will need to remove the six import at the beginning of this file